commands/host.vegman

```diff
@@ -63,6 +63,71 @@ function cmd_power_status {
   ipmitool power status
 }
 
+# @doc cmd_virtualmedia
+# Access Virtual Media
+function cmd_virtualmedia {
+  subcommand "$@"
+}
+
+# @sudo cmd_virtualmedia_mount admin
+# @doc cmd_virtualmedia_mount
+# Mount a virtual media
+#  --usb - the USB interface type
+#  --usb-ro - the USB interface type (read-only)
+#  --hdd - the HDD interface type
+#  --cdrom - the CDROM interface type (read-only)
+#  FILE — path to image file
+function cmd_virtualmedia_mount {
+  local imagefile=""
+  local type="usb"
+
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      --usb) type="usb";;
+      --usb-ro) type="usb-ro";;
+      --hdd) type="hdd";;
+      --cdrom) type="cdrom";;
+      *)
+        if [[ -n "${imagefile}" ]]; then
+          abort_badarg "$1"
+        else
+          imagefile="$1"
+        fi
+        ;;
+    esac
+    shift
+  done
+
+  if [[ ! -f "$imagefile" ]] && [[ ! -e "$imagefile" ]]; then
```

Comment:
In bash, you need quotes only for `-z` or `-n`, other cases are covered by `[[`.

Reply:
Fixed